server/admin/prompt.go

```diff
@@ -21,6 +21,19 @@ type passwordReadResponse struct {
 // PasswordPrompt prompts the user to enter a password. Password must not be an
 // empty string.
 func PasswordPrompt(ctx context.Context, prompt string) ([]byte, error) {
+	var password []byte
+	var err error
+	for len(password) == 0 {
+		password, err = passwordPrompt(ctx, prompt)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Can you comment on the desirability of a retry loop vs returning an error?

Reply:
> Can you comment on the desirability of a retry loop vs returning an error?
if users don't enter a password, we use the retry loop to give them another chance.